tcms_tenants/apps.py

```diff
@@ -0,0 +1,12 @@
+# pylint: disable=import-outside-toplevel
+from django.apps import AppConfig as DjangoAppConfig
+from django.core.checks import register
+
+
+class AppConfig(DjangoAppConfig):
+    name = "tcms_tenants"
```

Comment:
This attribute shouldn't be necessary. Try removing it and we'll test without it.

Reply:
An exception occurs without it:
django.core.exceptions.ImproperlyConfigured: 'tcms_tenants' must supply a name attribute.